docker/images/elasticsearch/5.6.16/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.16
+
+# Switch back to the root user to set permissions
+USER root:root
+
+# Make sure that the initial directories have the correct permissions
+# for OpenShift. OpenShift will run the elasticsearch with a user wit
+# a random UID and a GID 0.
+USER root
```

Comment:
Why switching back to `root` user twice?

Reply:
Because I was tired, I guess :man_facepalming: 
Thanks!